### DISCUSSION TITLE
I have a question regarding the CNN model I have prepared

### DISCUSSION THREAD

**Discussion by daniya-sohail26** [community/community#44187] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

! pip install -q kaggle
from google.colab import files
files.upload()
! mkdir ~/.kaggle
! cp kaggle.json ~/.kaggle/
! chmod 600 ~/.kaggle/kaggle.json
! kaggle datasets download -d tawsifurrahman/tuberculosis-tb-chest-xray-dataset
import zipfile
zf = "/content/tuberculosis-tb-chest-xray-dataset.zip"
target_dir = "/content/dataset/cnn/tuberculosis_revamped"
zfile = zipfile.ZipFile(zf)
zfile.extractall(target_dir)
 !pip install split-folders
import splitfolders 

splitfolders.ratio(
    "/content/dataset/cnn/tuberculosis_revamped/TB_Chest_Radiography_Database", 
    output="/content/dataset/cnn/tuberculosis", 
    seed=1337, 
    ratio=(.7, .2, .1), 
    group_prefix=None
)
!rm -r "/content/dataset/cnn/tuberculosis_revamped"
train_path = '/content/dataset/cnn/tuberculosis/train'
test_path = '/content/dataset/cnn/tuberculosis/test'
valid_path = '/content/dataset/cnn/tuberculosis/val'
how do I use this code in defining the path to predict image in pycharm interface? It would be a huge help if someone replies

**Comment by bzkarim250** [comment, marked as answer]
I got good answer for you on [ChatGpt](https://chat.openai.com/chat) , I think this may help you
